docs/.vuepress/public/_redirects

```diff
@@ -53,20 +53,100 @@
+/server/v20.10/security/trusted-intermediary.html                 /server/v20.10/security.html#trusted-intermediary  301!
+/server/v20.10/security/configuration.html                         /server/v20.10/security.html   301!
+/server/v20.10/indexes/advanced.html                             /server/v20.10/indexes.html   301!
+/server/v20.10/installation/kubernetes-aks.html                   /server/v20.10/installation.html   301!
+/server/v20.10/diagnostics/logging.html                            /server/v20.10/diagnostics.html   301!
+/server/v20.10/operations/scavenge-options.html                   /server/v20.10/operations.html#scavenging-events   301!
+/server/v20.10/projections/projections-config.html                 /server/v20.10/projections.html#configuring-projections   301!
+/server/v20.10/diagnostics/prometheus.html                         /server/v20.10/diagnostics.html#prometheus   301!
+
+
+
 ## Server version 5
 /server/v5/server/*                         /server/v5/:splat                          301!
 /server/v5.0/server/*                       /server/v5/:splat                          301!
 /server/5.0.8/server/*                      /server/v5/:splat                          301!
 /server/5.0.9/server/*                      /server/v5/:splat                          301!
-/clients/dotnet/5.0/connecting/options.html /clients/dotnet/5.0/connecting.html        301!
-
+/clients/dotnet/5.0/connecting/options.html /clients/dotnet/5.0                        301!
+
+/server/v5/server/default-directories.html                        /server/v5/server-settings.html#default-directories   301!
+/server/v5/server/database.html                                     /server/v5/server-settings.html#database-settings   301!
+/server/v5/server/introduction/                                   /server/v5/#introduction   301!
+/server/v5/introduction/clients.html                             /server/v5/#protocols-clients-and-sdks   301!
+/server/v5/server/projections/                                    /server/v5/projections.html   301!
+/server/v5/security/acl.html                                     /server/v5/security.html#access-control-lists   301!
+/server/v5/security/authentication.html                            /server/v5/security.html   301!
+/server/v5/security/configuration.html                            /server/v5/security.html   301!
+/server/v5/networking/                                           /server/v5/networking.html   301!
+/server/v5/networking/nat.html                                   /server/v5/networking.html#nat-and-port-forward   301!
+/server/v5/http-api/                                             /clients/http-api/v5/persistent.html   301!
+/server/v5/http-api/persistent/security.html                     /clients/http-api/v5/persistent.html   301!
+/server/v5/http-api/writing-events.html                          /clients/http-api/v5/#reading-streams-and-events   301!
```

Comment:
This seems a bit funny that the old link was about writing events and the redirect points to reading events?

Reply:
@cdevarenne That one might have been misplaced, I'll check it out!